libs/transactions/src/lib/components/transaction-item/transaction-item.component.html

```diff
@@ -35,3 +35,19 @@
   >
   </bb-amount-ui>
 </span>
+<span>
+  <button
+    bbButton
+    [color]="'primary'"
+    [buttonSize]="'sm'"
+    [routerLink]="[
+      '/transfer/make-transfer',
+      {
+        amount: transaction.transactionAmountCurrency.amount,
+        accountName: transaction.merchant?.name || transaction.counterPartyName
+      }
```

Comment:
Can we put these parameters between `transfer` and `make-transfer` to add more sense to use them?

Reply:
Attached a sceenshot

Reply:
Oh noo, you can do it by using `snapshot.parent`

Reply:
But your option is also nice

Reply:
But is it a good practice to use snapshot.parent in child components. If something needs to be used in the entire route hierarchy one should go for queryparams.

Reply:
Yes. It's alright if the parent route is a part of the journey